### ISSUE TITLE
Add bonuses from weapons

### ISSUE THREAD

**Issue by Unikatze** [jamespdaily/pf2e-sheet-skill-actions#19]
This may be a tricky one. 
But say I have a +1 Warhammer worn, which has the shove trait. 
Perhaps the module will add a toggle for me to add the +1 item bonus to my shove?

**Comment by andriusch** [comment]
One more thing to consider here - you might not have warhammer equipped. So either:
* We check if warhammer is equipped and add item bonus to shove
* Or we add a separate action for warhammer shove

I think first is preferrable, but probably harder to implement.

**Comment by Unikatze** [comment]
I'd go with the First Option. 

If you select Shove and you have a Warhammer equipped, it could just come up with a toggable Item bonus to the shove (default off).

**Comment by Glunty** [comment]
For your information, this feature is to be added in system action macros within the next release, with this MR : https://gitlab.com/hooking/foundry-vtt---pathfinder-2e/-/merge_requests/7260
But i don't see how we could easily get it added in this module :(

**Comment by andriusch** [comment]
As far as I understand it modifies `game.pf2e.actions` to add those bonuses. Since this module uses those actions under the hood everything should work automatically in this module too.

**Comment by Glunty** [comment]
Yep, should be good, we'll be sure on 11 Feb when new core version shall be released.

**Comment by jamespdaily** [comment]
Can I close this one in light of the Feb 11 release?

**Comment by Glunty** [comment]
I tested it with my branch on the system, it works. The only thing is that the item bonus is not showed on the button. But if it's a bug, it's quite a minor one

![image](https://user-images.githubusercontent.com/18362479/153272190-0065db15-e204-48ee-bc60-442709760157.png)

**Comment by andriusch** [comment]
Yeah, it's a bug since we calculate the roll modifier ourselves, not sure if it can be made to take it from PF2e. But yeah it's minor.

**Comment by Glunty** [comment]
I'm even sure it can't be taken from PF2e yet because that specific modifier is computed in an asynch function when a check is asked : even if you have the SimpleRollActionCheckOptions, you can't see weapon modifiers in it before the roll is called and message comes in the chat.

Can't think of a way to get them dynamically... perhaps by adding a RuleElement to all weapons with athletism traits, or something like that... won't be done soon I guess

**Comment by dapperdancer** [comment]
Is there also a way to integrate whether a weapon with a shove/trip/disarm trait is agile?
